packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/src/Utilities/StringExtensions.cs

```diff
@@ -1,15 +1,20 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
+using System.Linq;
 using System.Text;
+using System.Text.RegularExpressions;
```

Comment:
Lets convert this to shared source between the libraries in mgc.

Given they all ship together shared source should be fine and we don't have to worry about duplications